jackson/src/main/scala/io/finch/jackson/package.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2014, by Vladimir Kostyukov and Contributors.
+ *
+ * This file is a part of a Finch library that may be found at
+ *
+ *      https://github.com/finagle/finch
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributor(s): -
+ */
+
+package io.finch
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.finch.request.DecodeRequest
+import io.finch.response.EncodeResponse
+
+import scala.reflect.ClassTag
+
+package object jackson {
+
+  implicit def decodeJackson[A](implicit mapper: ObjectMapper, tag: ClassTag[A]) = new DecodeRequest[A] {
+    def apply(s: String): Option[A] = try {
+      val clazz = tag.runtimeClass.asInstanceOf[Class[A]]
+      Some(mapper.readValue[A](s, clazz))
```

Comment:
Check if it work for decode something that requires type erasure (`List[String]`)? 
ScalaObjectMapper uses a different method (`def readValue[T: Manifest](content: String): T`): 
https://github.com/FasterXML/jackson-module-scala/blob/master/src/main/scala/com/fasterxml/jackson/module/scala/experimental/ScalaObjectMapper.scala#L183